src/components/Tests/TestTicket/index.tsx

```diff
@@ -64,7 +65,7 @@ export const TestTicket = (props: TestTicketProps) => {
       description={{
         content: renderDescription()
       }}
-      attachments={[attachment]?.filter(Boolean).map((a) => a!)}
+      attachments={[attachment]?.filter((item): item is Attachment => !!item)}
```

Comment:
You can conditionally add the attachment to the array

```
  const attachments: Attachment[] = [
  ...(traceId
    ? [{
        url: `${config.jaegerURL}/api/traces/${traceId}?prettyPrint=true`,
        fileName: `trace-${traceId}.json`
      }]
    : [])
   ];
```
and avoid filtering

```
  attachments={attachments}
```

Reply:
done